src/lib/sidenav/drawer.ts

```diff
@@ -462,9 +490,15 @@ export class MatDrawerContainer implements AfterContentInit, OnDestroy {
 
       this._changeDetectorRef.markForCheck();
     });
+
+    this._debouncer.pipe(
+      debounceTime(10), // Arbitrary debounce time, less than a frame at 60fps
```

Comment:
is `debounce` what we want, doesn't `audit` make more sense here?

Reply:
I'd say so. `audit` just throttles the events, but what we want is to only measure once if multiple events happen very close to each other.

Reply:
If the drawer were animating open, wouldn't it just keep denouncing until the animation is over and then jump?

Reply:
AFAIK animations don't trigger change detection. `ngDoCheck` runs for every change detection cycle.